### ISSUE TITLE
Powershell loading Microsoft.Azure.Cosmos.ServiceInterop.dll fails

### ISSUE THREAD

**Issue by Ashish1662** [Azure/azure-cosmos-dotnet-v3#1313]
I am trying to access cosmos db sql api data in PS (windows 5.1 bcz the other module supports only this version).

I have created a custom class library .net v4.7.2 project which is using Microsoft.Azure.Cosmos as described [here](https://docs.microsoft.com/en-us/azure/cosmos-db/sql-api-get-started) to access db items.

When I try to load my custom dll in the PS it expects the dependencies too. And when I try to load Microsoft.Azure.Cosmos.ServiceInterop.dll from the bin folder, it always gives me error saying "**Could not load file or assembly 'file:///C:\Users\xxx\Desktop\PS\dlls\Microsoft.Azure.Cosmos.ServiceInterop.dll' or one of its dependencies. The module was expected to contain an assembly manifest.**"

![image](https://user-images.githubusercontent.com/49236220/77691195-809bea00-6fca-11ea-9573-60c4f578b99d.png)

I have tried setting the executing policy, unblocking the dlls etc...but nothing works.

Would appreciate if anyone could help.

**Environment summary**
SDK Version: 
.net framework v4.72.
Win PS 5.1
Microsoft.Azure.Cosmos

**Comment by NaluTripician** [comment]
Closing as part of a backlog cleanup pass.

**Reason:** Niche PowerShell load-failure scenario from 2020 with no comments, no repro on current SDK, and no active owner.

**Steps to reopen** - if this is still relevant on a current SDK release:
1. Add a comment with the SDK version you're on, current behavior, and a minimal repro (or updated requirements).
2. Reopen the issue (or ask someone with triage rights to reopen) and the team will re-triage.

cc/auto-triage: closing as `wontfix` per backlog hygiene; reopening is encouraged if circumstances have changed.
